jib-core/README.md

```diff
@@ -98,17 +98,42 @@ The Jib Core system consists 3 main parts:
 Some other parts of Jib Core internals include:
 
 - a caching mechanism to speed up builds (configurable with [`Containerizer.setApplicationLayersCache`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/Containerizer.html#setApplicationLayersCache-java.nio.file.Path-) and [`Containerizer.setBaseImageLayersCache`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/Containerizer.html#setBaseImageLayersCache-java.nio.file.Path-))
-- an eventing system to react to events from Jib Core during its execution (add handlers with [`Containerizer.setEventHandlers`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/Containerizer.html#setEventHandlers-com.google.cloud.tools.jib.event.EventHandlers-))
+- an [eventing system](#events) to react to events from Jib Core during its execution (add handlers with [`Containerizer.setEventHandlers`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/Containerizer.html#setEventHandlers-com.google.cloud.tools.jib.event.EventHandlers-))
 - support for fully-concurrent multi-threaded executions
 
+## Events
+
+Throughout the build process, Jib Core dispatches events that provide useful information. These events implement the type [`JibEvent`](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/event/JibEvent.html), and can be handled by registering event handlers with the event dispatcher that is passed to Jib.
+
+```java
+// Setup event handlers
+EventHandlers eventHandlers = new EventHandlers()
+    .add(JibEventType.LOGGING, logEvent -> System.out.println(logEvent.getLevel() + ": " + logEvent.getMessage()))
+    .add(JibEventType.TIMING, timeEvent -> ...);
+
+// Register with Jib
+Jib.from(...)
+    ...
+    .containerize(
+        Containerizer.to(...)
+            ...
+            .setEventHandlers(eventHandlers));
+```
+
+When Jib dispatches events, the event handlers you defined for that event type will be called. The following are the types of events you can listen for in Jib core (see [API reference](http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/package-summary.html) for more information):
```

Comment:
can this reference a *latest* instead of 0.1.1? Seems like we could possibly miss updating this when doing releases?

Reply:
Looks like the same thing through the readme.

Reply:
There appears to be some provision for it at: https://www.javadoc.io/, but I don't know how we're generating these in the first place.

Reply:
okay, so the url just has to be formatted a little weird

```diff
- http://static.javadoc.io/com.google.cloud.tools/jib-core/0.1.1/com/google/cloud/tools/jib/api/package-summary.html
+ https://www.javadoc.io/page/com.google.cloud.tools/jib-core/latest/com/google/cloud/tools/jib/api/package-summary.html
```

Reply:
Sounds like a good idea to me